bigbluebutton-tests/playwright/chat/chat.spec.js

```diff
@@ -13,6 +13,7 @@ test.describe('Chat', () => {
     const page = await context.newPage();
     await chat.initModPage(page, true);
     await chat.initUserPage(true, context);
+    await chat.initUserPage2(true, context)
```

Comment:
I believe we shouldn't initiate the second user here if we are not checking anything with it in the other tests. We should keep the user2 in this test but only there - at least for now